### ISSUE TITLE
Interactive elements not reachable with VoiceOver [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#264]
The “1-888-364-3065” button on the My Health Log Results C (“Not feeling well”) screen is not reachable with VoiceOver or navigating via keyboard alone. The button is grouped with its surrounding text and only the block of text can be focused, not the button it contains.

![](https://user-images.githubusercontent.com/4437084/92048361-88b83c80-ed55-11ea-99e6-5feb86e5f1dc.png)

The “Find a site near you” link on the How to Keep Others Safe screen suffers from the same problem.

![](https://user-images.githubusercontent.com/4437084/92048384-9372d180-ed55-11ea-8b14-30f251d52f52.png)

**Comment by floridemai** [comment]
@willslone  Can you check in the next build please. Thanks

**Comment by helloantoine** [comment]
This is sorted. Good job.
